stream/storage/api/src/main/java/org/apache/bookkeeper/stream/storage/api/metadata/RangeStoreService.java

```diff
@@ -22,4 +23,9 @@
     extends MetaRangeStore,
     RootRangeStore,
     TableStore {
```

Comment:
how about also add this in the interface comments in line 20?

Reply:
I think data includes "table".